CHANGELOG.md

```diff
@@ -69,6 +69,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 - server: fix importing of allow list query from metadata (fix #4687)
 - server: flush log buffer during shutdown (#4800)
 - server: fix edge case with printing logs on startup failure (fix #4772)
+- server: fix bug which arised when renaming a column if it was used to create the remote relationship (fix #4995)
```

Comment:
i think we can ignore the changelog as it is a bug fix on master (i.e. unreleased version)